### PR TITLE
Fix grammar error

### PR DESCRIPTION
**Pull request by tswicegood**

### CODE REVIEW

File: bokeh/charts/_chart.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def show(self):
         elif self._options.filename is False and \
                         self._options.server is False and \
                         self._options.notebook is False:
-            print("You have a provide a filename (filename='foo.html' or"
+            print("You have to a provide a filename (filename='foo.html' or"
```
Comment (bryevdv) [inline bokeh/charts/_chart.py]: As long as we are changing this I much prefer "You must provide a filename..."

Comment (tswicegood) [inline bokeh/charts/_chart.py]: Yeah, that reads better.  Just tried to limit my change to make it correct with the smallest change.

File: bokeh/charts/_chart.py [hunk 2]
```diff
@@ -248,7 +248,7 @@ def show(self):
         elif self._options.filename is False and \
                         self._options.server is False and \
                         self._options.notebook is False:
-            print("You have a provide a filename (filename='foo.html' or"
+            print("You must a provide a filename (filename='foo.html' or"
```
Comment (bryevdv) [inline bokeh/charts/_chart.py]: OK but now there is an extra 'a' :)

Comment (tswicegood) [inline bokeh/charts/_chart.py]: I've been a space cadet all day... :space_invader:

**Comment by fpliger** [comment]
LGTM

**Comment by bryevdv** [comment]
Not going to waste a travis test cycle on this, merging.

**Pull request merged**